keps/sig-apps/0753-sidecarcontainers.md

```diff
@@ -82,6 +84,24 @@ Check these off as they are completed for the Release Team to track. These check
 
 To solve the problem of container lifecycle dependency we can create a new class of container: a "sidecar container" that behaves primarily like a normal container but is handled differently during termination and startup.
 
+## Prerequisites
+
+On June 23 2020, during SIG-node meeting, it was decided that this KEP has a
+prerequisite on the (not yet submitted KEP) kubelet node graceful shutdown.
+
+As of writing, when a node is shutdown the kubelet doesn't shutdown gracefully
+all the containers (running preStop hooks and other guarantees user expect). It
+was, then, decided that adding more guarantees to pod shutdown behavior (as this
+KEP proposes) depends on having the kubelet gracefully shutdown first. The main
+reason for this is to avoid users relying on something (like the pod shutdown
+sequence) we can't guarantee.
+
+Also, authors of this KEP and the (yet not submitted) KEP for node graceful
+shutdown have met several times and are in sync regarding these features
+interoperability.
+
+The dependecy is proposed in the appha/beta/stable graduation.
```

Comment:
dependecy -> dependency
appha -> alpha

Reply:
Thanks! Updated it to link to the section too, so it is easier to navigate :)

(will push in a few minutes)